Makefile

```diff
@@ -60,7 +60,7 @@ podman_cleanup:
 bell:
 	@echo "Done!" $$'\a'
 
-assisted_deployment: requirements configure
+assisted_deployment:
```

Comment:
Looks fine but can you split the commit message, the "they were moved ..." should go in the description rather than the very long title

Reply:
Ok I fixed this myself so we can merge